examples/js/postprocessing/MaskPass.js

```diff
@@ -32,6 +32,11 @@ THREE.MaskPass.prototype = {
 		state.setColorWrite( false );
 		state.setDepthWrite( false );
```

Comment:
Yes. `setColorWrite` and `setDepthWrite` disable the depth buffer for writing for masking. The following statements `setColorBuffer` and `setDepthBuffer` ensures, that no subsequent calls of `WebGLRenderer .setMaterial` overwrite this setting.